.env

```
@@ -0,0 +1,7 @@
+# JWT Secret Key
+JWT_SECRET=tHG2nqQbANXZC/IxjIwd7ItBCFAEhdD+KdFxkLyoGq7wkeZ3TGdwPhlMxd66dNlF
+           WY0RYxT36vcx8oM0x+Zlnw
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The WY0RYxT36vcx8oM0x+Zlnw key has incorrect delimiter

(IncorrectDelimiter)